Pipfile.lock

```diff
@@ -0,0 +1,597 @@
+{
```

Comment:
Do we really need this file? What's a Pipfile.lock? Sounds like a temporary file.

Reply:
Pipenv is a dependency manager that is much more flexible than pip, with built-in virtualenv integration and lock file generation. The idea is that `Pipfile` documents ranges of package versions the project is supposed to work with, and `Pipfile.lock` contains what specific versions are currently being used, so everyone in the team can use the exact same versions to ensure consistent behavior. It can also generate a `requirements.txt` with locked versions for people still using `pip`. Package revision updates will change `Pipfile.lock` and `requirements.txt`, while updates to backwards-incompatible versions will require changes to the code as well as to `Pipfile` to document the new compatible ranges.

All 3 files should be committed to source control, to ensure everyone has the same environment.